`make compose-build` fails

I've been able to run `make build` succesfully, but `make compose-build` results in this error:

```
go fmt plugin/loader/preload.go >/dev/null
#9 30.46 go build "-asmflags=all='-trimpath=/go'" "-gcflags=all='-trimpath=/go'" -ldflags="-X "github.com/ipfs/go-ipfs".CurrentCommit=64b532f" -o "cmd/ipfs/ipfs" "github.com/ipfs/go-ipfs/cmd/ipfs"
#9 53.36 /bin/sh: 1: /build/cleanup.sh: not found
------
executor failed running [/bin/sh -c apt-get update &&     apt-get install -y build-essential git &&     git clone --depth 1 --branch $IPFS_VERSION https://github.com/ipfs/go-ipfs.git /go-ipfs &&     cd /go-ipfs && make build &&     /build/cleanup.sh]: exit code: 127
ERROR: Service 'ipfs-node-1' failed to build : Build failed
make: *** [compose-build] Error 1
```

--- a/deployments/Dockerfile.daemon
+++ b/deployments/Dockerfile.daemon
@@ -7,8 +7,7 @@ ENV IPFS_VERSION v0.10.0
 RUN apt-get update && \
     apt-get install -y build-essential git && \
     git clone --depth 1 --branch $IPFS_VERSION https://github.com/ipfs/go-ipfs.git /go-ipfs && \
-    cd /go-ipfs && make build && \
-    /build/cleanup.sh
+    cd /go-ipfs && make build
 
 COPY cmd/ipfs-daemon /go/cmd/ipfs-daemon
 COPY pkg /go/pkg

--- a/deployments/Dockerfile.monitor
+++ b/deployments/Dockerfile.monitor
@@ -3,8 +3,7 @@ FROM golang:1.17.10-bullseye as builder
 WORKDIR /go
 
 RUN apt-get update && \
-    apt-get install -y build-essential && \
-    /build/cleanup.sh
+    apt-get install -y build-essential
 
 COPY cmd/ipfs-gw-measure /go/cmd/ipfs-gw-measure
 COPY cmd/ipfs-gw-monitor /go/cmd/ipfs-gw-monitor

